packages/edit-site/src/components/dataviews/view-actions.js

```diff
@@ -22,11 +21,11 @@ import { __ } from '@wordpress/i18n';
 import { unlock } from '../../lock-unlock';
 
 const {
-	DropdownMenuV2,
-	DropdownMenuGroupV2,
-	DropdownMenuItemV2,
-	DropdownSubMenuV2,
-	DropdownSubMenuTriggerV2,
+	DropdownMenuV2Ariakit,
+	DropdownMenuGroupV2Ariakit,
+	DropdownMenuItemV2Ariakit,
+	DropdownMenuRadioItemV2Ariakit,
+	DropdownMenuCheckboxItemV2Ariakit,
```

Comment:
It feels a bit weird to me that these components have Ariakit in their names, I understand that they're still private so we can change.

Reply:
Absolutely 💯  I plan on renaming those components once the radix-ui version of the dropdownmenu [gets removed](https://github.com/WordPress/gutenberg/pull/55626)